### DISCUSSION TITLE
Director-based Workflow or Aggregator-based Workflow ?

### DISCUSSION THREAD

**Discussion by dskhanirfan** [FETS-AI/Challenge#140] [General]
Which FL based workflow is deployed in FeTS Director-based Workflow or Aggregator-based Workflow ? 

Ref:
[Director-based Workflow](https://openfl.readthedocs.io/en/latest/source/workflow/director_based_workflow.html) 
[Aggregator-based Workflow](https://openfl.readthedocs.io/en/latest/source/workflow/running_the_federation.agg_based.html)

**Comment by sarthakpati** [comment]
Updated references:
- https://openfl.readthedocs.io/en/latest/running_the_federation.html?highlight=workflow#director-based-workflow
- https://openfl.readthedocs.io/en/latest/running_the_federation.html?highlight=workflow#aggregator-based-workflow

**Reply by dskhanirfan** [reply]
@sarthakpati which one is used in FeTS?

**Reply by sarthakpati** [reply]
I believe we are doing Aggregator-based workflow.

**Comment by msheller** [comment]
Yeah, we used an Aggregator-based workflow for the FeTS initiative, and the FeTS challenge used a single-process simulation, which is suitable for testing new FL algorithms on a single machine.
